rnd/autogpt_server/autogpt_server/data/db.py

```diff
@@ -0,0 +1,28 @@
+import asyncio
+
+from uuid import uuid4
+from prisma import Prisma
+from pydantic import BaseModel
+
+prisma = Prisma(auto_register=True)
+
+
+def connect_sync():
+    asyncio.get_event_loop().run_until_complete(connect())
+
+
+async def connect():
+    await prisma.connect()
+
+
+async def disconnect():
+    await prisma.disconnect()
```

Comment:
This is redundant especially outside class imo, you can just directly call `prisma.connect()`

Reply:
Why is it redundant ?

Reply:
It's ok, that function is used outside, my bad